src/components/test/common-tests.js

```diff
@@ -96,7 +96,7 @@ export function testPresentationalComponent(Component, componentName) {
       assert.isTrue(wrapperOuterEl.hasAttribute('data-component'));
       assert.equal(
         wrapperOuterEl.getAttribute('data-component'),
-        componentName ?? Component.name
+        componentName ?? Component.displayName ?? Component.name
```

Comment:
Is this still needed, given that the component's `name` properties haven't changed?

Reply:
Well-spotted! Technically _no_, but I believe it's the right precedence overall.